crates/red_knot_python_semantic/src/module_resolver/resolver.rs

```diff
@@ -36,18 +38,18 @@ pub(crate) fn resolve_module_query<'db>(
     let name = module_name.name(db);
     let _span = tracing::trace_span!("resolve_module", %name).entered();
 
-    let Some((search_path, module_file, kind)) = resolve_name(db, name) else {
+    let Some((search_path, module)) = resolve_name(db, name) else {
```

Comment:
Would it be slightly clearer to call this `resolved` instead of `module`? On first reading I was confused about `ResolvedModule` type vs `Module` type created below.